jargons/babel.md

```diff
@@ -0,0 +1,8 @@
+---
+title: babel
+tags:
+- javascript
+- ecmascript
+---
+
+Henüz resmi olarak gelmemiş [ECMAScript](/ecmascript) özelliklerini önceki sürümlerde de kullanmanızı sağlayan, [transpile](/transpile) yöntemi ile çalışan bir araçtır.
```

Comment:
@f 

> Tarayıcıların henüz resmi olarak desteklemediği [ECMAScript](/ecmascript) özelliklerini kullanabilmek için kullandığımız bir araçtır. Tarayıcının desteklemediği standartlara yeni eklenen özellikleri [transpile](/transpile) yöntemi ile tarayıcıların desteklediği sürüme dönüştürür.

gibi bir aciklama daha iyi olabilir sanki, pek toparlayamadim gerci ama :/

Reply:
Transpile işini ayrı olarak açıklarız diye düşündüm.

Reply:
Bir de "tarayıcının" demedim zira Node da var.

Reply:
"Derleyici" bu is icin dogru bir kavram mi?  Gereksiz yere mi isi karistiriyor. 

Bu isleri yapmak cok zor gercekten 😄